app/views/conversations/show.html.erb

```diff
@@ -20,15 +20,13 @@
     <% end %>
   </div>
 
-  <ul role="list" class="mt-8 space-y-2">
+  <ul id="messages" role="list" class="mt-8 space-y-2">
```

Comment:
`<ul>` actually comes with an implicit `role` of `"list"` so you dont need to specify it again! (it can sometimes even cause unexpected behavior when you add a role to an element that has the same implicit role)

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/list_role

Reply:
Huh! TIL, thank you!